packages/create-ripple-app/tidemodules.js

```diff
@@ -12,6 +12,7 @@ module.exports = [
   { value: 'webform', name: 'Webform', default: true },
   { value: 'search', name: 'Search', default: true },
   { value: 'publication', name: 'Publication', default: false },
+  { value: 'dataDrivenComponent', name: 'Data Driven Component', default: false },
```

Comment:
@dylankelly this module has been removed from Ripple to https://github.com/dpc-sdp/yourvictoria-vic-gov-au/tree/develop/packages/data-driven-component. So no need to have it in config anymore.

Reply:
Ahh ok will remove